ci/tests/test_geneformer/test_geneformer_model.py

```diff
@@ -45,4 +47,11 @@ def test_ensembl_data_is_caught(self, gene_symbols, raises_error):
         if raises_error:
             with pytest.raises(ValueError):
                 self.geneformer.process_data(self.data, "gene_symbols")
+
+    def test_fine_tune_classifier(self):
+        assert self.geneformer.fine_tune_classifier is not None
+        fine_tuned_model = self.geneformer.fine_tune_classifier(self.tokenized_dataset)
+        assert fine_tuned_model is not None
+        assert fine_tuned_model is not self.geneformer.model
+        assert isinstance(fine_tuned_model, BertForSequenceClassification)
```

Comment:
Are you sure this is checking that the weights of the fine-tuned model have been updated indeed? got a feeling that is might only be checking the class (on line 55 i mean). Might be wrong though.

Reply:
It doesn't do a weight check. At the moment it is just checking that it is not the original model, that we have instantiated a new model for the fine tuning and that the return type is correct.